lock_test.go

```diff
@@ -115,6 +115,24 @@ func TestLockExclusive(t *testing.T) {
 	if err != lock.ErrAlreadyLocked {
 		t.Errorf("err = %s, expected the lock to fail due to the resource already being locked", err)
 	}
+
+	// Create a lock with some resource name and some lockId
+	// that resource expires after some time
+	// then with same resource name and lockId can able to create lock.
+	err = client.XLock("resource4", "cccc", lock.LockDetails{TTL: 2})
```

Comment:
To keep the tests as short as possible, can you make the TTL `1` here?

Reply:
Yup, the testcase time should be as short as possible.